tests/ctst/features/resource-policies/regen.js

```diff
@@ -152,21 +152,26 @@ const output = scenarios.map(scenario => {
     const paddedBucketPolicyApplies = scenario.bucketPolicyApplies.padEnd(longest.bucketPolicyApplies);
     const paddedBucketPolicyEffect = scenario.bucketPolicyEffect.padEnd(longest.bucketPolicyEffect);
 
+    // eslint-disable-next-line max-len
```

Comment:
shouldn't we reformat instead:

```suggestion
return (   
	`           `, paddedAction, paddedBucketPolicyExists. paddedBucketPolicyApplies,
	paddedBucketPolicyEffect, paddedIamPolicyExists, paddedIamPolicyApplies, paddedIamPolicyEffect,
).join(' | )
```